content/docs/command-reference/pull.md

```diff
@@ -217,6 +220,11 @@ $ dvc remote list
 r1	ssh://_username_@_host_/path/to/dvc/remote/storage
 ```
 
+With the first `dvc pull` we specified a stage in the middle of this pipeline
+(`matrix-train.p.dvc`) while using `--with-deps`. DVC started with that `.dvc`
+file and searched backwards through the pipeline for data files to download.
+Because the `model.p.dvc` stage occurs later, its data was not pulled.
+
```

Comment:
Bad conflict resolution. This paragraph is repeated now (see line 205). Please just remove these lines. Thanks

Reply:
I'll do it:
```suggestion
```